xbmc/games/addons/GameClientProperties.h

```diff
@@ -57,13 +57,13 @@ class CGameClientProperties
   const char** GetProxyDllPaths(void);
 
   // Number of proxy DLLs needed to load the game client
-  unsigned int GetProxyDllCount(void) const { return m_proxyDllPaths.size(); }
+  size_t GetProxyDllCount(void) const { return m_proxyDllPaths.size(); }
```

Comment:
Can you drop these? I fixed this in an upcoming comment by moving the both to the .cpp file.